node/config/def.go

```diff
@@ -58,16 +58,17 @@ func DefaultBoost() *Boost {
 	cfg := &Boost{
 		Common: defCommon(),
 
+		MpoolAlertEpochs: 30,
```

Comment:
Because all the other config fields are under a section, we should probably move this one under a section too.
I suggest we call the section "Monitoring"

Reply:
Added the new section.

```
./boostd config default --no-comment | grep -A1 Monitoring
[Monitoring]
  MpoolAlertEpochs = 30
```